tutorials/kb_guided_llm_ner/spacy.cfg

```diff
@@ -0,0 +1,19 @@
+[nlp]
+lang = "en"
+pipeline = ["llm","dbpedia-spotlight"]
+
+[components]
+
+[components.llm]
+factory = "llm"
+
+[components.llm.task]
+@llm_tasks = "spacy.NER.v2"
+labels = ["Person", "Organisation"]
+
+[components.llm.model]
+@llm_models = "spacy.GPT-4.v1"
+config = {"temperature": 0.3}
+
+[components.dbpedia-spotlight]
+factory = "dbpedia_spotlight"
```

Comment:
No worries if you don't know, but it is possible to configure this pipeline for [non-English](https://github.com/MartinoMensio/spacy-dbpedia-spotlight#using-dbpedia-in-a-specific-language) as well?

Reply:
Yes, (good point!) `spacy-dbpedia-spotlight` exposes this setting as part of its factory config. Added a param on the CLI of this recipe and a note in README!